src/PKSim.Infrastructure/Serialization/Xml/Serializers/ApplicationSettingsXmlSerializer.cs

```diff
@@ -14,6 +14,7 @@ public ApplicationSettingsXmlSerializer()
 
       public override void PerformMapping()
       {
+         Map(x => x.MoBiPath);
```

Comment:
user is not required to set the path. If using the portable setups
The suite does not have portable options. For now, portable only available for PKSim and MoBi